hugr-core/src/hugr/validate/test.rs

```diff
@@ -926,6 +929,13 @@ fn cfg_children_restrictions() {
     b.remove_node(exit2);
 
     // Change the types in the BasicBlock node to work on qubits instead of bits
+    b.replace_op(
```

Comment:
this test failed because it was getting away with not updating the cfg op beforehand